run.sh

```diff
@@ -9,6 +9,8 @@ export PATH=/root/bin:$PATH
 #Path to k8s config file
 KUBECONFIG=/home/blockchain-automation-framework/build/config
 
+echo "Validatin network yaml"
+ajv validate -s /home/blockchain-automation-framework/platforms/network-schema.json -d /home/blockchain-automation-framework/build/network.yaml 
```

Comment:
This looks good but it uses another tool that has to be installed by the user. Did you check ansible assert module?

Reply:
Included also validations with ansible assert module and sent pros/cons of each by mail for your evaluation.